frontend/interfaces/host.ts

```diff
@@ -87,10 +87,10 @@ export interface IMunkiData {
 }
 
 export interface IHostMdmData {
-  enrollment_status: string;
+  enrollment_status: MdmEnrollmentStatus;
   server_url: string;
-  id: number;
-  name: string;
+  id?: number;
+  name?: string;
```

Comment:
@RachelElysia We saw you added `id` and `name` to this interface but couldn't find them being used and they aren't returned by the API. Is that something we need?